qa/TL3_RN50_convergence/test_mxnet.sh

```diff
@@ -9,7 +9,7 @@ NUM_GPUS=`nvidia-smi -L | wc -l`
 sed -i "s/nvJPEGDecoder/ImageDecoder/" /opt/mxnet/example/image-classification/common/dali.py
 
 python /opt/mxnet/example/image-classification/train_imagenet_runner \
-       --data-root=/data/imagenet/train-val-recordio-passthrough/ -b 208 \
+       --data-root=/data/imagenet/train-val-recordio-passthrough/ -b 180 \
```

Comment:
This fails with bigger batch size with updated MXNet?

Reply:
Yes, it is OOM with 21.04.